packages/app-project/src/components/AuthModal/components/RegisterForm/components/Form/Form.js

```diff
@@ -71,7 +71,6 @@ function Form ({ errors, handleBlur, handleChange, handleSubmit, isSubmitting, v
             required
           >
             <TextInput
-              autoFocus
```

Comment:
I've assigned this PR to myself to review this week. Out of curiosity, was `autoFocus` removed in AuthModal due to lint warning?

Reply:
Yes. It actually stops the NextJS apps from building, because `next build` halts on ESLint errors.

Reply:
It’s also a big accessibility problem.